subworkflows/local/longread_hostremoval.nf

```diff
@@ -54,7 +54,7 @@ workflow LONGREAD_HOSTREMOVAL {
 
     emit:
     stats    = SAMTOOLS_STATS.out.stats     //channel: [val(meta), [reads  ] ]
-    reads    = SAMTOOLS_BAM2FQ.out.reads   // channel: [ val(meta), [ reads ] ]
+    reads    = SAMTOOLS_FASTQ.out.other   // channel: [ val(meta), [ reads ] ]
```

Comment:
Not that we use these files anywhere else, I think it would be safer to `mix` all of teh outputs from `FASTQ`, not just other? This would have the same behavior as the old module - otherwise here you might lose of the other reads I think (depending on how `samtools fastq` interprets the input, no?)

Reply:
Do you suggest that we have 

`
reads = SAMTOOLS_FASTQ.out.fastq?`

Reply:
I see, that makes sense because `samtools/bam2fq` had one emit with all reads. 
I will test and update the PR.